src/site/_site/Preview/index.js

```diff
@@ -18,6 +24,26 @@ export default class Preview extends React.Component {
     }));
   }
 
+  renderFullPageLink() {
+    if (this.state.isDemo) {
```

Comment:
should this be `!this.state.isDemo`?  Not seeing the link on the main page.

Reply:
no - it's only for the demo pages right now